Warning about "shapper" package for the class

alt2177 opened the pull request:
It's not worth the headache I promise you.

alt2177 committed:
Warning about "shapper" package for the class

It's not worth the headache I promise you.

jtr13 commented on shap.Rmd:
```diff
@@ -27,3 +27,5 @@ ex.
 ### Examples
 
+
+**NOTE: DO NOT USE THE `SHAPPER` PACKAGE. IT IS AWFUL**
```
Can you be more specific about what the issue is?

alt2177 replied:
tl;dr About a 2.36/10 (and that’s being generous). The dependencies are no longer available on CRAN and the example code provided does not work (see [here](https://github.com/ModelOriented/shapper))

More specifically, the fact that this package requires you to install Python dependencies to then wrap R around those Python functions seems rather silly. The only possible use I could see for this is if you are a Python developer who refuses to learn R and needs to use shapper. But at that point, why not do the project in Python itself? If you have to migrate to R, there are a lot of other SHAP packages available that work great (see shapr for example).

If you insist on using this package, I am sorry for your loss and hope you don’t have too many hours of unpaid overtime left. Godspeed ahead.

alt2177 committed:
Added more detailed review of "shapper"

I hope this is enough of an explanation as to why "shapper" should never be used.
